experiment/build/docker_images.py

```diff
@@ -50,6 +55,11 @@ def get_images_to_build(fuzzers, benchmarks):
     for fuzzer in fuzzers:
         for benchmark in benchmarks:
             for name_templ, obj_templ in templates.items():
+                if 'coverage' in fuzzer and 'runner' in name_templ:
+                    continue
+                if 'base' in name_templ or 'dispatcher' in name_templ:
+                    images[name_templ] = obj_templ
+                    continue
```

Comment:
what'd happen if we removed this if branch?

Reply:
The previous image set (separate oss-fuzz and standard) returned this as a list for these, so subtitute would fail. I can try that again and change if it doesn't return a list anymore.